tests/Avalonia.Animation.UnitTests/AnimatableTests.cs

```diff
@@ -36,7 +36,7 @@ public void Transition_Is_Not_Applied_When_Not_Attached_To_Visual_Tree()
         [Fact]
         public void Transition_Is_Not_Applied_To_Initial_Style()
         {
-            using (UnitTestApplication.Start(TestServices.RealStyler))
+            using (Start())
```

Comment:
Previously these tests passed because a null clock was being passed to a mock, where if it were to be passed to a real object it would have thrown.